src/components/package/edit-custom/custom-package-edit.js

```diff
@@ -290,25 +304,30 @@ class CustomPackageEdit extends Component {
                       id="packageInfo"
                       onToggle={this.toggleSection}
                     >
-                      {packageSelected ? (
-                        <NameField />
-                      ) : (
-                        <KeyValue label={<FormattedMessage id="ui-eholdings.package.name" />}>
-                          <div data-test-eholdings-package-readonly-name-field>
-                            {model.name}
-                          </div>
-                        </KeyValue>
-                      )}
-
-                      {packageSelected ? (
-                        <ContentTypeField />
-                      ) : (
-                        <KeyValue label={<FormattedMessage id="ui-eholdings.package.contentType" />}>
-                          <div data-test-eholdings-package-details-readonly-content-type>
-                            {model.contentType}
-                          </div>
-                        </KeyValue>
-                      )}
+                      {packageSelected
+                        ? (
+                          <NameField />
+                        )
```

Comment:
Single component could written without brackets 
```suggestion
                      {packageSelected
                        ? <NameField />
                        : (
```